test/integration/server.coffee

```diff
@@ -31,6 +31,7 @@ describe 'Server', ->
 
   describe 'Deployed API', ->
     before (done)->
+      process.env['CLOUDMINE'] = 1
```

Comment:
This is setting to a number where prior checks look for truthy? I'm guessing 1 is truthy?